src/main/resources/pid.properties

```diff
@@ -63,12 +63,16 @@ database.sqlserver=pid-3569588c-b89d-5567-84ee-a2c633c7204c
 
 dynamic.aad.end=pid-f5a60b13-efd6-551a-a40f-3923804e475d
 dynamic.aad.start=pid-7dcc8904-9a8a-5b71-a73d-6caa9edb78ef
+dynamic.addnode.start=pid-5f68f633-701b-58b0-83c2-399c6d1d8414
+dynamic.addnote.end=pid-449e677a-5fc3-55cb-9193-4b225f42e646
```

Comment:
Hello @edburns  we have to add those pids to oracle publisher in partner center, to track adding nodes to dynamic cluster.